### PR TITLE
feat: add the ability to set serviceAccount annotations for PrometheusOperator

### PR DESCRIPTION
**Pull request by fernandezcuesta**
Signed-off-by: Jesús Fernández <7312236+fernandezcuesta@users.noreply.github.com>

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 62.3.0
+version: 62.3.1
```
Comment (GMartinez-Sisti) [inline charts/kube-prometheus-stack/Chart.yaml:26]: Hi @fernandezcuesta, can you please bump minor for feature instead?
```suggestion
version: 62.4.0
```

**Comment by jkroepke** [comment]
For consistency, could you add annotation for prometheus-operator/admission-webhooks/, too:

https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/deployment/serviceaccount.yaml

File: charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/job-patch/serviceaccount.yaml [hunk 2]
```diff
@@ -10,6 +10,9 @@ metadata:
   labels:
     app: {{ template "kube-prometheus-stack.name" $ }}-admission
     {{- include "kube-prometheus-stack.prometheus-operator-webhook.labels" $ | nindent 4 }}
+  {{- with .Values.prometheusOperator.serviceAccount.annotations }}
```
Comment (jkroepke) [inline charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/job-patch/serviceaccount.yaml:13]:
```suggestion
  {{- with .Values.prometheusOperator.admissionWebhooks.patch.serviceAccount.annotations }}
```

File: charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/deployment/serviceaccount.yaml [hunk 3]
```diff
@@ -8,6 +8,9 @@ metadata:
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-operator
     {{- include "kube-prometheus-stack.prometheus-operator-webhook.labels" . | indent 4 }}
+  {{- with .Values.prometheusOperator.serviceAccount.annotations }}
```
Comment (jkroepke) [inline charts/kube-prometheus-stack/templates/prometheus-operator/admission-webhooks/deployment/serviceaccount.yaml:11]:
```suggestion
  {{- with .Values.prometheusOperator.admissionWebhooks.deployment.serviceAccount.annotations }}
```

**Review by jkroepke — APPROVED** [review]
Thanks! please also add the annotations values to the values.yaml

**Review by jkroepke — APPROVED** [review]

**Pull request merged**